mmedit/models/builder.py

```diff
@@ -1,24 +1,11 @@
-import torch.nn as nn
-from mmcv import build_from_cfg
+from mmcv.cnn import build_model_from_cfg
+from mmcv.utils import Registry
 
-from .registry import BACKBONES, COMPONENTS, LOSSES, MODELS
+MODELS = Registry('models', build_func=build_model_from_cfg)
```

Comment:
You may refer to [MMDet example](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/builder.py#L6) to set `parent` argument with MMCV_MODELS.